README.md

```diff
@@ -66,24 +66,31 @@ Purser has three components to install.
 
 #### Purser Setup
 ``` bash
+# Namespace setup
+kubectl create ns purser
```

Comment:
Why do we need to create it in a particular name? 
Also, if this is a limitation, for now, add an explanation of why we need to create a namespace, if this is necessary and what will happen if it created in any other namespace.